examples/interaction/tooltips/tooltip_description.py

```diff
@@ -1,9 +1,19 @@
+''' A visualization of Multichoice tooltips in bokeh.models.
+This example demonstrates user input text box where
+the user can input from al list of various options of elements to choose one or more of them,
+and then the clear the choices as well.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.io.show, bokeh.models.MultiChoice, Bokeh.models.Tooltip
+    :refs: `ug_interaction_widgets_examples_multichoice`
+    :keywords: tooltip, multichoice
+'''
```

Comment:
one blank line after metadata block (please update other docstrings as well)
```suggestion
    :keywords: tooltip, multichoice
    
'''
```